src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -300,13 +303,11 @@ void FixedwingAttitudeControl::Run()
 			parameters_update();
 		}
 
-		/* only run controller if attitude changed */
-		static uint64_t last_run = 0;
-		float deltaT = constrain((hrt_elapsed_time(&last_run) / 1e6f), 0.01f, 0.1f);
```

Comment:
Note: this was wrong, the FW attitude controller typically runs faster than the minimum interval here.